### DISCUSSION TITLE
12-12

### DISCUSSION THREAD

**Discussion by ngsy** [ChristmasTest/christmas#74] [Daily_Meeting]
### 🎅🏻 회의 내용
- 작업한 내용 공유 
윤혁님 :  Footer 컴포넌트
수님 : 테스트 페이지, 결과 페이지
수영님 : Return 컴포넌트
슬희님 : 오류 해결!

- 역할 분담 
노션봐!

### ❄️ 오늘 할일

1. 강의 듣기 
2. 운동

### ⛄️ TMI
졸려요.. 잘자요~

**Comment by oauch** [comment]
저도 졸려요 ➰
